1주차_baekjoon_1550

Centslabs opened the pull request:
🎁week01-2 과제 완료 :)

Centslabs committed:
Create README.md

Centslabs committed:
Create W01-1_자기소개.py

자기소개 :)

Centslabs committed:
Create W01-2_baekjoon1550(ValueError).py

백준에 채점시 런타임 에러 (ValueError) 발생.
이유가 무엇일까요?..

hyuk3880 reviewed (commented):
소문자를 대문자로 바꿔야 돌아가는거 같습니다~

Centslabs committed:
W01_2_baekjoon1550 완료

16진수는 대문자만 허용한다 ★★★★★
숏코드 print(int(input(), 16))

Centslabs commented:
1550번 16진수 변환 완료 :)
소문자가 문제 였네요 :)

Centslabs committed:
Create W01-3_baekjoon_10872.py

백준 10872 백토리얼 완료 :)
입력이 1보다 작을 때 결과는 항상 1 :)

minji0401 reviewed (approved):
주석 되게 깔끔해요!! 😆

Centslabs committed:
W01_4_baekjoon_1032 완료

프린트는 join이 간결하지만 속도는 for문이 빠르다.
기준값인 fileNames[0]을 리스트로 감싸줘야 한다.

Centslabs committed:
W01_5_baekjoon_10828 완료

중요 포인트!
input() 함수를 사용할 경우, 시간초과 에러가 발생.
시간단축을 위해 sys.stdin.readline()을 사용한다.
입출력 속도 비교 : sys.stdin.readline > raw_input() > input()

Centslabs committed:
W01_7_baekjoon_10845 완료

Centslabs commented:
과제 완료 :)
